core/metacling/src/TCling.cxx

```diff
@@ -1684,6 +1684,19 @@ static const std::unordered_set<std::string> gIgnoredPCMNames = {"libCore",
                                                                  "G__GenVector32",
                                                                  "G__Smatrix32"};
 
+static void MaybePrintDlError(const char* dyLibName, const char* modulename) {
```

Comment:
Why is the routine prefixed by 'Maybe'?

Reply:
Good point, that's not the right name anymore.